doc/changelog/README.md

```diff
@@ -31,7 +31,8 @@ manual for details.
 Here is a summary of the structure of a changelog entry:
 
 ``` rst
-- Description of the changes, with possible link to
+- **Added / Changed / Fixed ...**
```

Comment:
If the list in the script is meant to be exhaustive it should be reproduced here instead of using ...
If not then it should have a way to input a custom category somehow.

Also I don't really like the past participle for this, I'd prefer nouns ie `Addition / Change / Fix / ...` especially considered you put a `.` after it.
In other words I could see `Changed:\n bla` or `Change.\n bla` but `Changed.\n bla` feels wrong.

Reply:
The list is intended to be exhaustive and the types are meant to be exactly those defined in the Keep a Changelog 1.0 specification (so keeping Added instead of Addition). I like your suggestion to use a colon instead of a dot though. Will fix, as well as give the exhaustive list here.

Reply:
>The list is intended to be exhaustive and the types are meant to be exactly those defined in the Keep a Changelog 1.0 specification

I see you said that in the OP post and commit message now that I look at them, maybe it should be put in the readme too.

Reply:
Fixed.

Reply:
Fixed. Maybe the mention of the reasons for not including the "Security" type is too long though.